k8s/k8s.go

```diff
@@ -122,7 +153,7 @@ func (k8s *Client) NamespaceByName(namespaceName string) (*v1.Namespace, error)
 }
 
 // NewClient returns a new kubernetes client.
-func NewClient(host, token, nodeName string, insecure bool) (*Client, error) {
+func NewClient(host, token, nodeName string, insecure, dealWithDupIP bool) (*Client, error) {
```

Comment:
`s/dealWithDupIP/allowMultiplePODs/g` ?  I'm not sure why there would be value in having this behavior be conditional.  Why not always search for pods?